res/controllers/Denon-MC7000-scripts.js

```diff
@@ -37,22 +37,35 @@ var MC7000 = {};
 // 1) Beat LED in Slicer mode (counts every 8 beats AFTER the CUE point)
 //    only works for CONSTANT TEMPO tracks
 //    needs beat grid and CUE point set
-MC7000.experimental = false;
+MC7000.experimental = false; // default: false
```

Comment:
redundant // redundant

;-)

Reply:
nope - not really - i have put the default value to each user variable so the user who modified it will know what the delivered setting was

Reply:
They can do that themselves though when modifying it. Or look it up in the manual or in the original file...

Reply:
Or they can add the comment themselves when they change it.

Reply:
I wrote those defaults to user variables without you claiming this was an issue before. As your reviews are not consistent over commits it is very hard to understand your point of view each time again...

Reply:
I agree with Jan and Niko. Please remove the redundant default values from comments. They are confusing and this is not common practice.

If users decide to modify the file they can decide how to preserve the original values if needed, e.g. duplicate and comment the original line, use Git or whatever.

You could add a comment at the beginning of the section that the following variables have been introduced to allow for some degree of customization if needed.